pkg/ring/kv/client_test.go

```diff
@@ -0,0 +1,31 @@
+package kv
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+func TestParseConfig(t *testing.T) {
+	conf := `
+store: consul
+consul:
+  host: "consul:8500"
+  consistentreads: true
+prefix: "test/"
+multi:
+  primary: consul
+  secondary: etcd
+`
+
+	cfg := Config{}
+
+	err := yaml.Unmarshal([]byte(conf), &cfg)
+	require.NoError(t, err)
+	require.Equal(t, "consul", cfg.Store)
+	require.Equal(t, "test/", cfg.Prefix)
+	require.Equal(t, "consul:8500", cfg.Consul.Host)
+	require.Equal(t, "consul", cfg.Multi.Primary)
+	require.Equal(t, "etcd", cfg.Multi.Secondary)
+}
```

Comment:
Could we have a couple of tests for bad config too? Like not setting secondary, setting both to the same thing etc.

Reply:
This test doesn't actually run any new code, it is just doing YAML parsing. Reason is that by extracting `StoreConfig` from `Config` struct, I broke YAML parsing of existing configurations, and wanted to make sure it's ok now. I can add tests for config verification as well, but it seems like overkill.